cg/apps/loqus.py

```diff
@@ -46,7 +46,8 @@ def load(
             "--hard-threshold": str(hard_threshold) if hard_threshold else None,
             "--soft-threshold": str(soft_threshold) if soft_threshold else None,
         }
-        load_call_params: list = ["load"] + get_list_from_dictionary(load_params)
+        load_call_params: List[str] = ["load"] + get_list_from_dictionary(load_params)
+        load_call_params.append("--qual-gq") if qual_gq else None
```

Comment:
To be consistent with the existing pattern, maybe this line can be moved into the dictionary above?

Reply:
It's not possible since this key is a flag and the previous method `get_list_from_dictionary` would remove it as it would have an empy value